src/analysis/normed_space/lp_space.lean

```diff
@@ -505,4 +582,127 @@ end
 
 end normed_space
 
+section topology
+
+open filter
+open_locale topological_space uniformity
+
+/-- The coercion from `lp E p` to `Π i, E i` is uniformly continuous. -/
+lemma uniform_continuous_coe [_i : fact (1 ≤ p)] : uniform_continuous (coe : lp E p → Π i, E i) :=
+begin
+  have hp : p ≠ 0 := (ennreal.zero_lt_one.trans_le _i.elim).ne',
+  rw uniform_continuous_pi,
+  intros i,
+  rw normed_group.uniformity_basis_dist.uniform_continuous_iff normed_group.uniformity_basis_dist,
+  intros ε hε,
+  refine ⟨ε, hε, _⟩,
+  rintros f g (hfg : ∥f - g∥ < ε),
+  have : ∥f i - g i∥ ≤ ∥f - g∥ := norm_apply_le_norm hp (f - g) i,
+  exact this.trans_lt hfg,
+end
+
+lemma norm_apply_le_of_tendsto {C : ℝ} {F : ℕ → lp E ∞} (hCF : ∀ᶠ k in at_top, ∥F k∥ ≤ C)
+  {f : Π a, E a} (hf : tendsto (id (λ i, F i) : ℕ → Π a, E a) at_top (𝓝 f)) (a : α) :
+  ∥f a∥ ≤ C :=
```

Comment:
Can the results in this topology section be generalized beyond `ℕ` and the `at_top` filter?